### PR TITLE
Raven db 12673 cert pass

### PR DESCRIPTION
**Pull request by iftahbe**

### CODE REVIEW

File: src/Raven.Server/Utils/Cli/RavenCli.cs [hunk 1]
```diff
@@ -825,26 +825,46 @@ private static bool CommandReplaceClusterCert(List<string> args, RavenCli cli)
...
             try
             {
                 certBytes = File.ReadAllBytes(path);
-                cert = new X509Certificate2(path, password, X509KeyStorageFlags.Exportable | X509KeyStorageFlags.PersistKeySet | X509KeyStorageFlags.MachineKeySet);
+                cert.Import(certBytes, password, X509KeyStorageFlags.Exportable | X509KeyStorageFlags.MachineKeySet);
             }
             catch (Exception e)
             {
-                WriteError("Failed to load the provided certificate. Please check the path and password." + e, cli);
+                throw new ArgumentException("Failed to load the provided certificate. Please check the path and password.", e);
```
Comment (ppekrol) [inline src/Raven.Server/Utils/Cli/RavenCli.cs]: Don't we need to use `WriteError` here?

File: src/Raven.Server/Utils/Cli/RavenCli.cs [hunk 2]
```diff
@@ -825,26 +825,46 @@ private static bool CommandReplaceClusterCert(List<string> args, RavenCli cli)
...
+            if (string.IsNullOrWhiteSpace(password) == false)
+            {
+                try
+                {
+                    // With the private key, without the password
+                    certBytes = cert.Export(X509ContentType.Pkcs12);
+                }
+                catch (Exception e)
+                {
+                    throw new ArgumentException("Failed to export the password provided certificate.", e);
```
Comment (ppekrol) [inline src/Raven.Server/Utils/Cli/RavenCli.cs]: Don't we need to use WriteError here?

Comment (ppekrol) [reply src/Raven.Server/Utils/Cli/RavenCli.cs]: And return `false` ?

File: src/Raven.Server/Utils/Cli/RavenCli.cs (hunk 1 above)
Comment (ppekrol) [reply src/Raven.Server/Utils/Cli/RavenCli.cs]: And return `false` ?

**Review by ppekrol — APPROVED** [review]

**Review by ayende — APPROVED** [review]

**Pull request merged**